common/lib/capa/capa/templates/jsinput.html

```diff
@@ -1,11 +1,12 @@
+<%page expression_filter="h"/>
 <%! from openedx.core.djangolib.markup import HTML %>
-<section id="inputtype_${id}" class="jsinput"
+<div id="inputtype_${id}" class="jsinput"
```

Comment:
Changed section to div because section is not really appropriate here (and caused a11y issue due to lack of header).